src/gui/guiFormSpecMenu.cpp

```diff
@@ -784,16 +784,20 @@ void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &el
 {
 	std::vector<std::string> parts = split(element, ';');
 
-	if (parts.size() != 3 &&
-			!(parts.size() > 3 && m_formspec_version > FORMSPEC_API_VERSION)) {
-		errorstream << "Invalid animated image element(" << parts.size()
-				<< "): '" << element << "'"  << std::endl;
+	if (parts.size() != 6 && parts.size() != 7 &&
+		!(parts.size() > 4 && m_formspec_version > FORMSPEC_API_VERSION))
```

Comment:
```suggestion
		!(parts.size() > 7 && m_formspec_version > FORMSPEC_API_VERSION))
```

Reply:
I find this code sooooo ugly.  Putting the brace on the next line look so much cleaner and easier to understand, and it's what `GUIFormSpecMenu.cpp` usually does.  Oh well.

Reply:
Sorry, I couldn't resist doing the style "fixes" in the suggestion. The important part is the `7`.